### DISCUSSION TITLE
Add option (checkbox) to disable ALT + X for making clips

### DISCUSSION THREAD

**Discussion by tetraDB** [Chatterino/chatterino2#3336] [Ideas] [answered]
Been making loads of random clips until I realised this was the reason, would be great to have a checkbox that disables this feature.

**Comment by Felanbird** [comment, marked as answer]
This will be possible with #2340
![image](https://i.imgur.com/pbbg1Hr.gif)

**Reply by tetraDB** [reply]
Great news, thanks!
